app/models/name.rb

```diff
@@ -1639,22 +1679,33 @@ def self.parse_author(str)
   end
 
   def self.parse_group(str, deprecated = false)
-    results = nil
-    if match = GROUP_PAT.match(str)
-      name = match[1]
-      text_name = name.tr("ë", "e")
-      parent_name = name.sub(LAST_PART, "")
-      results = ParsedName.new(
-        text_name: text_name + " group",
-        search_name: text_name + " group",
-        sort_name: format_sort_name(text_name, "group"),
-        display_name: format_name(name, deprecated) + " group",
-        parent_name: parent_name,
-        rank: :Group,
-        author: ""
-      )
+    return unless match = GROUP_PAT.match(str)
+
+    # Parse the string without "group"
+    ungrouped_str = str.sub(/\s#{GROUP_ABBR}/, "")
```

Comment:
Should this have a slash-b at the end of the regex to prevent premature match of, say, a species epithet which begins with "group" (or later "clade")?